dashboard/src/components/AppView/AppView.test.tsx

```diff
@@ -114,4 +122,36 @@ describe("AppViewComponent", () => {
       expect(sockets.length).toEqual(0);
     });
   });
+
+  describe("renderization", () => {
+    it("renders all the elements of an application", () => {
+      const wrapper = shallow(<AppViewComponent {...validProps} />);
+      expect(wrapper.find(ChartInfo).exists()).toBe(true);
+      expect(wrapper.find(DeploymentStatus).exists()).toBe(true);
+      expect(wrapper.find(AppControls).exists()).toBe(true);
+      expect(wrapper.find(ServiceTable).exists()).toBe(true);
+      expect(wrapper.find(AppNotes).exists()).toBe(true);
+      expect(wrapper.find(AppDetails).exists()).toBe(true);
+    });
+    it("renders a forbidden error if it exists", () => {
```

Comment:
These last two tests are testing the displaying of the deleteError, what about if there is an error when initially fetching the application?